share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -107,39 +56,38 @@ class Foo < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-debug", "--disable-dependency-tracking"
-#   system "cmake", ".", *std_cmake_args
+    # system "cmake", ".", *std_cmake_args
     system "make", "install"
   end
 end
 ```
 
-**Note:**  If `brew` said `Warning: Version cannot be determined from URL` when doing the `create` step, you’ll need to explicitly add the correct version to the formula with `version "foo"` **and then save the formula**. `brew install` should then proceed without any trouble.
-
-**Note:** If `brew` said `No formula found for "php54-timezonedb". Searching open pull requests...` and you are writing a Tap, you should run `brew tap --repair`.
+If `brew` said `Warning: Version cannot be determined from URL` when doing the `create` step, you’ll need to explicitly add the correct [`version`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#version-class_method) to the formula and then save the formula.
 
-**Note:** Homebrew will try to guess the formula’s name from its URL. If it
-fails to do so you can use `brew create <url> --set-name <name>`.
+Homebrew will try to guess the formula’s name from its URL. If it fails to do
+so you can override this with `brew create <url> --set-name <name>`.
 
-## Fill in the Homepage
+## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
 
-**We don’t accept formulae without homepages!**
+**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
 
-SSL/TLS (https) homepage is preferred, if one is available.
+A SSL/TLS (https) [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method) is preferred, if one is available.
 
-Homebrew now has a description field (`desc`). Try and summarize this from the homepage.
+Try to summarize from the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method) what the formula does in the [`desc`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#desc%3D-class_method)ription.
```

Comment:
Maybe worth mentioning the `desc` automatically outputs the formula name and consequently it doesn't need including? Dunno. It's audit flagged at least if we omit it here.